gradle.properties

```diff
@@ -9,9 +9,9 @@ org.gradle.daemon=false
   fabric_version=0.43.1+1.18
 
 # Mod Properties
-	group=com.sollace
-	displayname=Copper Golem
-	authors=Sollace
+    group=com.sollace
+    displayname=Copper Golem
+    authors=Sollace
```

Comment:
This is a "tab to space" change; let me know if that matters.

Reply:
Should be fine. Thanks!